src/applications.md

```diff
@@ -2,22 +2,28 @@
 
 Applications of Rust:
 * [Book of crosvm](https://google.github.io/crosvm/) - a hosted (a.k.a. type-2) virtual machine monitor
+* Embedded
+  * [The Rust on ESP Book](https://esp-rs.github.io/book/)
 * Game development
   * [Amethyst Game Engine](https://book.amethyst.rs/stable/)
   * [DMG-01: How to Emulate a Game Boy](https://rylev.github.io/DMG-01/public/book/)
   * [Emergent AI](https://psichix.github.io/emergent/) - smart agents and events
   * [Roguelike Tutorial - In Rust](https://bfnightly.bracketproductions.com/) - [Roguelike](https://en.wikipedia.org/wiki/Roguelike) game development
   * [The Specs Book](https://specs.amethyst.rs/docs/tutorials/) - an introduction to [Entity–component–system (ECS)](https://en.wikipedia.org/wiki/Entity_component_system) and the Specs API
   * [Writing NES Emulator in Rust](https://bugzmanov.github.io/nes_ebook/index.html) - NES is the Nintendo Entertainment System gaming platform
-* GPU Development
+* GPU development
   * [Rust GPU Dev Guide](https://embarkstudios.github.io/rust-gpu/book/)
 * GUI development
   * [GUI development with Relm4](https://aaronerhardt.github.io/relm4-book/book/)
   * [GUI development with Rust and GTK 4](https://gtk-rs.org/gtk4-rs/stable/latest/book/)
   * [SixtyFPS Memory Game Tutorial (Rust)](https://sixtyfps.io/releases/0.1.4/docs/tutorial/rust/) - tutorial using the SixtyFPS GUI toolkit
 * [intermezzOS OS](http://intermezzos.github.io/book/second-edition/)
 * [Juniper - GraphQL Server for Rust](https://graphql-rust.github.io/juniper/current/)
-* [mdBook](https://rust-lang.github.io/mdBook/)
 * [Prusti user guide](https://viperproject.github.io/prusti-dev/user-guide/) - a Rust verifier
+* [Serde](https://serde.rs/) - **ser**ialize and **de**serialize Rust data structures
```

Comment:
This book is already in the unofficial chapter. However, since it isn't in mdbook format - see the [source](https://github.com/serde-rs/serde-rs.github.io), it should move to `other-formats.md`. I'm becoming less and less fond of the division into mdbook format and other formats, so let's leave it in the unofficial chapter for now and implement https://github.com/lborb/book/issues/29 sooner rather than later.

Reply:
I pulled this from unofficial and moved it into applications as part of this proposed change as I don't see `serde` documentation as being about the Rust lang since it is a framework.